lib/utils/note-title.js

```diff
@@ -10,14 +10,29 @@
 const noteTitleAndPreviewRegExp = /(\S[^\n]*)\s*(\S[^\n]*)?/;
 
 /**
+ * Matches the title and excerpt in a note's content skipping opening # from title
+ *
+ * Both the title and the excerpt are determined as
+ * content starting with something that isn't
+ * whitespace and leads up to a newline or line end
+ * @type {RegExp} matches a title and excerpt in note content skipping opening # from title
+ */
+const noteTitleAndPreviewMdRegExp = /[# ]*(\S[^\n]*)\s*(\S[^\n]*)?/;
+
+/**
  * Returns the title and excerpt for a given note
  *
  * @param {Object} note a note object
  * @returns {Object} title and excerpt (if available)
  */
 export const noteTitleAndPreview = note => {
   const content = note && note.data && note.data.content;
-  const match = noteTitleAndPreviewRegExp.exec(content || '');
+  const isMarkdown =
+    note && note.data && note.data.systemTags.some(tag => tag === 'markdown');
```

Comment:
totally fine here as-is, but this makes me think we need to get together a function like `isMarkdown( note )` since that's becoming an important test condition.